tests/pyaaas/models/test_privacy_models_LDiversity.py

```diff
@@ -0,0 +1,38 @@
+from pyaaas.models.privacy_models import LDiversityDistinct, LDiversityGrassbergerEntropy, LDiversityShannonEntropy, LDiversityRecursive
+from pyaaas.aaas import AaaS
+import unittest
+
+class PrivacyModelsTest(unittest.TestCase):
```

Comment:
Burde ikke denne hete LDiversityDinstinctTest?

Reply:
Split tests into multiple files, and refactored the names